src/libraries/System.Reflection/tests/MethodInfoTests.cs

```diff
@@ -894,8 +894,26 @@ private static void SecondCall(MethodInfo mi)
             Assert.Contains("TestAssembly", asm.ToString());
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/71095", TestRuntimes.Mono)]
```

Comment:
I think this method is missing `[Fact]` attribute.

Reply:
Thanks